crates/runtimelib/src/jupyter/kernelspec.rs

```diff
@@ -31,7 +31,7 @@ impl KernelspecDir {
         Ok(spec.clone())
     }
 
-    pub async fn command(
+    pub fn command(
```

Comment:
I noticed this function wasn't async at all and really shouldn't be, so this will end up becoming a `0.23.0` release.